result_quality.md

```diff
@@ -0,0 +1,56 @@
+# Determining Result Quality
+
+Pelias tries very hard to always return good results, and generally does a pretty good job. If using the autocomplete endpoint or otherwise showing results directly to humans, judging result quality is generally easy (if subjective).
+
+However, when programmatically managing results from Pelias, determining result quality is much harder. Pelias returns several different properties with each result to help determine if a result is good enough for your purposes.
+
+
+### layer
+This is essentially what type of result was returned. Examples are address, venue, neighbourhood, city, state. The `layers` parameter can be used to filter on this if you know ahead of time what you want. Results returned that are of a different layer than you expect generally mean incorrect results.
```

Comment:
Could go for air-quotes again here for 'type'?

I would describe it more like:

"Layers are used to partition source data, they are analogous to a 'table' in a relational database model'

"In the Pelias model. layers generally define the 'granularity' of the result, from 'country' to 'neighbourhood' down to 'venue' and 'address'.

The `layers` parameter can be used to filter on this if you know ahead of time what you want.

Careful filtering of layers allows you to 'pick and choose' which results are returned.